.github/ISSUE_TEMPLATE/1-O3R-ISSUE-FORM.yaml

```diff
@@ -24,6 +25,7 @@ body:
         configuration,
         core,
         create,
+        design,
         dev-tools,
```

Comment:
we should add telemetry as well.
Ideally we should find a way to ensure we keep this up to date.

Reply:
![Screenshot 2024-03-18 at 11 33 39](https://github.com/AmadeusITGroup/otter/assets/2467012/5b3fda31-15e7-478c-a246-38d7fafb2447)